### PR TITLE
[FEATURE] improve header display

### PR DESCRIPTION
**Pull request by darrensiegel**
This PR adds a separator between the header and the content below it. It also reduces the padding in the header .

### CODE REVIEW

File: assets/styles/authoring/layout/workspace.scss [hunk 1]
```diff
@@ -25,9 +25,14 @@
 
   .workspace-header {
     width: 100%;
-    padding: 12px 47px 6px 47px;
+    padding: 6px 10px 3px 20px;
```
Comment (eliknebel) [inline assets/styles/authoring/layout/workspace.scss:28]: With this padding change, the live view loading indicator now lays on top of the account icon

loading

**Comment by dsuriano** [comment]
Tested locally and everything looks fine, but it struck me as odd that `Help` and the new User Account icon are only present inside a project and not globally. 

Any particular reason for that?

**Comment by dsuriano** [comment]
Trying to retest but getting this error: 

image

**Comment by darrensiegel** [comment]
I've backed out the changes that attempted to update all headers.  Now the "workspace" and "project" headers both display the Help and Account links.

**Review by dsuriano — APPROVED** [review]
Tested locally and works as expected.

**Review by eliknebel — APPROVED** [review]

**Pull request merged**